src/app/modules/calendar/calendar.component.html

```diff
@@ -0,0 +1,85 @@
+<form #f="ngForm" (input)="validateDate()" attr.data-auto-id="{{ dataAutoId }}">
+  <legend class="ds-c-label--error">
+    <span *ngIf="!valid" class="ds-c-field__hint ds-u-color--error">
+      {{ errorMessage }}
+    </span>
+  </legend>
+  <div class="ds-l-form-row">
+    <div class="ds-l-col--auto">
+      <label
```

Comment:
If possible, it woudl be a good idea to use the textfield component here, so that the styles are 100% consistent with other form inputs

Reply:
I am using a specific material design directive [matDatepicker]="datePicker". Don't know if that's something we can use or worth adding to texfield component